metadata/extra-animations.xml

```diff
@@ -153,6 +153,14 @@
 					<value>3</value>
 					<_name>Bottom</_name>
 				</desc>
+				<desc>
+					<value>4</value>
+					<_name>Scroll X</_name>
```

Comment:
Maybe we can name these Horizontal Scroll and Vertical Scroll or something? Not a huge deal, but Scroll X/Y isn't super descriptive. Also, we might need to change the `carpet_roll_direction` option long description, to make it more obvious what these options do.